client/src/components/DataFiles/DataFilesModals/DataFilesShowPathModal.jsx

```diff
@@ -55,36 +54,16 @@ const DataFilesShowPathModal = React.memo(() => {
         className="dataFilesModal"
       >
         <ModalHeader toggle={toggle} charCode="&#xe912;">
-          Pathnames for {file.name}
+          View Full Path
         </ModalHeader>
         <ModalBody>
-          <DataFilesBreadcrumbs
-            api={params.api}
-            scheme={params.scheme}
-            system={params.system}
-            path={file.path || '/'}
-            section=""
-          />
-          <dl>
-            {params.api === 'tapis' && definition && (
-              <>
-                <dt>Storage Host</dt>
-                <dd>{definition.host}</dd>
-                <dt>Storage Path</dt>
-              </>
-            )}
-            {params.api === 'googledrive' && (
-              <>
-                <dt>Storage Location</dt>
-                <dd>Google Drive</dd>
-              </>
+          <TextCopyField
+            value={`${definition.host}:${definition.rootDir}/${file.path}`.replace(
+              '//',
+              '/'
```

Comment:
@wesleyboar  do we need to combine the path and system into one box? The user would likely be more interested in copying the path without the system prefix there. Might be better keeping system and path boxes separate

Reply:
@rstijerina @wesleyboar Should we just have the system listed above it (not copyable?) And the full path (copyable) in its own box?

Reply:
IMO the system ID text should still be selectable, just in a different box, and doesn't need a copy button for the system ID

Reply:
<img width="1428" alt="modalv1" src="https://github.com/TACC/Core-Portal/assets/50084480/83dd5596-f091-4a42-a0cc-04b824baf11e">
<img width="1428" alt="modalv2" src="https://github.com/TACC/Core-Portal/assets/50084480/15e33aa8-0233-410a-85cd-e59660e64f9c">
Personally I like the top version more, but up to y'all.

Reply:
Agreed. As far as the storage path text area, is it that large by design @wesleyboar ?

Reply:
Yes. Please talk to C.J. about this. He may just need you to correct his assumption of how user uses the data. I'll start the conversation in chat.

Reply:
I'll let him review, or you can summarize it in [thread](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1696521997902929).